src/pages/docs/mcp_server/overview.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Overview
+description: Introduction to MCP Server capabilities and integration
+---
+
+{% mcp-blur %}
+
+## What is MCP Server?
```

Comment:
We do not have any MCP server deployed anywhere. Please remove from the pull request and include it in a different pull request for when we have one

Reply:
Please remove